linera-core/src/client/mod.rs

```diff
@@ -361,8 +359,8 @@ impl MessagePolicy {
         }
     }
 
-    #[instrument(level = "trace", skip(self))]
-    fn handle(&self, bundle: &mut IncomingBundle) -> bool {
+    #[tracing::instrument(level = "trace", skip(self))]
```

Comment:
I saw a recent PR doing the opposite `tracing::instrument -> instrument`

Reply:
Oh, I didn't notice this change - will revert.